langchain/src/schema/runnable/utils.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * An object that can be added to another object.
+ */
+export class AddableObject {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  data: Record<string, any>;
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  constructor(initialData: Record<string, any> = {}) {
+    this.data = initialData;
+  }
+
+  add(other: AddableObject): AddableObject {
```

Comment:
Python has `add` and `radd` because Python supports dunder methods:

https://mathspp.com/blog/pydonts/dunder-methods

We've been using `.concat` in JS:

https://github.com/langchain-ai/langchainjs/blob/a5d0ccacb84377783b003ce6529144b6cc1945f3/langchain/src/schema/index.ts#L190

Reply:
Resolved by removing `add` and `radd` with a `concat`